src/network_systems/projects/local_transceiver/src/local_transceiver_ros_intf.cpp

```diff
@@ -135,7 +139,10 @@ int main(int argc, char * argv[])
 {
     bool err = false;
     rclcpp::init(argc, argv);
+  
+    // std::shared_ptr<LocalTransceiver> lcl_trns = std::make_shared<LocalTransceiver>("", SATELLITE_BAUD_RATE);
     try {
+        //std::shared_ptr<LocalTransceiverIntf> node = std::make_shared<LocalTransceiverIntf>(lcl_trns);
```

Comment:
could you tell me the reason why you commented the local transciever constructor out? looking at `local_transciever.h`, the constructor still requires `port_name` and `baud_rate`

Reply:
Hi Sam, I believe Henry wrote most of this, but from what I remember, this change is because the LocalTransceiverIntf class used to take an instance of Local Transceiver as a parameter for its constructor, so one was created in main to be passed into the function. However, later on this might have been changed to that the LocalTransceiverIntf constructor makes its own instance of Local Transceiver, so it no longer required one to be passed as a parameter, so it was commented out. I forgot to remove it as a parameter in the documentation and delete the commented statements.